docs/api/visualizer/create.md

```diff
@@ -0,0 +1,178 @@
+---
+id: create
+title: Create a new Visualizer
+sidebar_label: Create
+keywords:
+  - OpenHIM
+  - API
+  - Visualizer
+  - Create
+description: Create an OpenHIM Visualizer via the API
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Create visualizer
+
+To create a new visualizer record you will need to make a TLS request to the OpenHIM API for the below method and endpoint and supply the JSON object for the visualizer record.
+
+
+```curl
+Method: POST
+Endpoint: /visualizers
+Payload: JSON object of the visualizer record
+```
+
+## Example
+
+<Tabs
+  defaultValue="lang"
+  values={[
+    { label: 'Languages', value: 'lang' },
+    { label: 'Sample Data', value: 'sample' }
+  ]
+}>
+<TabItem value="lang">
+
+  Before we can send our request to the OpenHIM API we need to ensure that we construct our valid HTTP headers to successfully authenticate with the OpenHIM API. 
+
+  <Tabs
+    defaultValue="nodejs"
+    values={[
+      { label: 'NodeJS', value: 'nodejs' },
+      { label: 'Bash', value: 'bash' }
+    ]
+  }>
+  <TabItem value="nodejs">
+
+  Copy the below code at the bottom of your nodejs script that handles the authentication of the OpenHIM headers as described in the [authentication section](../introduction/authentication.md). 
+
+  Replace the `openhimOptions` values with the correct implementation details and supply the `jsonData` payload to submit
+
+  ```javascript
+  // append below code to the "openhim-api.js" script containing the authentication methods. 
+  // This is described within the authentication section
+
+  (async () => {
+    const openhimOptions = {
+      apiURL: 'https://localhost:8080',
+      apiEndpoint: '/visualizers',
+      username: 'root@openhim.org',
+      password: 'openhim-password',
+      rejectUnauthorized: false
+    }
+    const jsonData = 'jsonData'
+
+    const headers = await genAuthHeaders(openhimOptions)
+    
+    const options = { 
+      method: 'POST',
+      url: `${openhimOptions.apiURL}${openhimOptions.apiEndpoint}`,
+      rejectUnauthorized: openhimOptions.rejectUnauthorized,
+      headers: headers,
+      body: jsonData,
+      json: true
+    }
+    
+    request(options, (error, response, body) => {
+      if (error) throw new Error(error)
+    
+      console.log({
+        statusCode: response.statusCode,
+        body
+      })
+    })
+  })()
+  ```
+
+  Execute the below command in your terminal to run the nodejs script
```

Comment:
@BMartinos Perhaps we could include a note to say what directory the user should be in to run this?

Reply:
done in 5a34e40